SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/ErrorAttachmentLog.cs

```diff
@@ -1,18 +1,83 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Microsoft.AppCenter.Ingestion.Models;
+using Newtonsoft.Json;
+using System.Text;
+
 namespace Microsoft.AppCenter.Crashes
 {
-    public partial class ErrorAttachmentLog
+    /// <summary>
+    /// Error attachment log.
+    /// </summary>
+    [JsonObject("errorAttachment")]
+    public partial class ErrorAttachmentLog : Log
     {
+        private const string ContentTypePlainText = "text/plain";
+
         static ErrorAttachmentLog PlatformAttachmentWithText(string text, string fileName)
         {
-            return new ErrorAttachmentLog();
+            var data = Encoding.UTF8.GetBytes(text);
```

Comment:
Shouldn't we check for null text? [Encoding.UTF8.GetBytes](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getbytes?view=netframework-4.8#System_Text_Encoding_GetBytes_System_String_) will throw an exception if `text` is null.

Reply:
Yeah it I added the null check on another branch. Should appear here shortly.